tests/e2e/e2e_suite_test.go

```diff
@@ -94,7 +56,7 @@ func deployOperator(k *kubectl.Kubectl) {
 		err = k.WaitForPod("cattle-rancheros-operator-system", "app=rancheros-operator", "rancheros-operator")
 		Expect(err).ToNot(HaveOccurred())
 
-		waitNamespace("cattle-rancheros-operator-system", "app=rancheros-operator")
+		k.WaitForNamespaceWithPod("cattle-rancheros-operator-system", "app=rancheros-operator")
```

Comment:
I've already added a `err` check, will be pushed with the commit that will upgrade `ele-testhelpers` version.